pkgs/misc/pylode/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, buildPythonApplication, fetchFromGitHub, rdflib, rdflib-jsonld, requests, dateutil, markdown, jinja2, isodate, gunicorn, falcon, pytest, six }:
+
+buildPythonApplication rec {
+  pname = "pyLODE";
+  version = "2.8.3";
+
+  src = fetchFromGitHub {
+    owner = "RDFLib";
+    repo = pname;
+    rev = version;
+    sha256 = "01a6b916r32irnllfr0rxz2mn9v6xn94lrgjdb8dhkki9ninqddy";
+  };
+
+  # Waiting for argparse to be removed in requirements.txt from upstream package
+  postPatch = ''
+    substituteInPlace requirements.txt --replace "argparse" ""
+  '';
+
+  propagatedBuildInputs = [
+    rdflib
+    rdflib-jsonld
+    requests
+    dateutil
+    markdown
+    jinja2
+    gunicorn
+    pytest
+    falcon
+    isodate
+    six
+  ];
+
+  # Import errors in pylode tests
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/RDFLib/pyLODE";
+    description = "An OWL ontology documentation tool using Python and templating, based on LODE";
+    license = licenses.gpl3;
```

Comment:
Use of this is deprecated per https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206

It looks like ``gpl3Only`` vs ``gpl3Plus`` is appropriate here based on the license text.